src/api/spec/models/bs_request_spec.rb

```diff
@@ -565,4 +565,28 @@
       end
     end
   end
+
+  describe '#as_json' do
+    before do
+      submit_request.update(superseded_by: delete_request.id)
+    end
+
+    subject { submit_request.as_json }
+
+    it 'returns a json representation of a bs request' do
+      expect(subject['id']).to eq(submit_request.id)
```

Comment:
For testing hashes, I find the include matcher easier to comprehend:

```
-      expect(subject['id']).to eq(submit_request.id)
-      expect(subject['number']).to eq(submit_request.number)
+      expect(subject).to include('id' => submit_request.id, 'number' => submit_request.number)
```

Just an idea

Reply:
Nice, I'll do that

Reply:
As one-liner it looks like 
```it { is_expected.to include('id' => submit_request.id, 'number' => submit_request.number) }```

which is pretty fancy - as it ouputs:
```
BsRequest
  #as_json
    should include {"id" => 1, "number" => 1}
```

rspec is pretty neat :)